vunit/nvc_interface.py

```diff
@@ -51,7 +51,8 @@ def setup_library_mapping(self, project, vhdl_standard):
         """
         Setup the library mapping according to project
         """
-
+        project.add_external_library("ieee_proposed",
```

Comment:
Agreed. A hint in the documentation should do the job. The actual issue here is that I have `ieee_proposed` installed by the `nvc` helper script already, but when I use `run.py` `ieee_proposed` is not found anymore.